[2024-07-22T02:34:14.875+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-09-15T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:34:14.943+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-09-15T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:34:14.949+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:34:14.955+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:34:14.964+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:34:15.018+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-09-15 00:00:00+00:00
[2024-07-22T02:34:15.027+0000] {standard_task_runner.py:55} INFO - Started process 2382 to run task
[2024-07-22T02:34:15.039+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-09-15T00:00:00+00:00', '--job-id', '819', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpl1xi989t', '--map-index', '1']
[2024-07-22T02:34:15.045+0000] {standard_task_runner.py:83} INFO - Job 819: Subtask add_one
[2024-07-22T02:34:15.202+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 819 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-09-15T00:00:00+00:00/task_id=add_one/map_index=1/attempt=1.log'; 2382)
[2024-07-22T02:34:15.233+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:34:15.380+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
